src/source/CppMarshal.scala

```diff
@@ -61,9 +61,7 @@ class CppMarshal(spec: Spec) extends Marshal(spec) {
     case MList => List(ImportRef("<vector>"))
     case MSet => List(ImportRef("<unordered_set>"))
     case MMap => List(ImportRef("<unordered_map>"))
-    case MCallback1 | MCallback2 | MCallback3 | MCallback4 | MCallback5 | MCallback6 | MCallback7
-       | MCallback8 | MCallback9 | MCallback10 | MCallback11 | MCallback12 | MCallback13
-       | MCallback14 | MCallback15 => List(ImportRef("<functional>"))
+    case MCallback2 => List(ImportRef("<functional>"))
```

Comment:
This is not needed anymore as the `callback2` type will get replaced by a `{List,}Callback` version.